src/ListView/ListView.stories.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { storiesOf, action } from '@kadira/storybook'
+import { withKnobs, boolean } from '@kadira/storybook-addon-knobs'
+import { defaultTemplate } from '../../storybook/decorators/storyTemplates'
+import MockListManager from './__mocks__/MockListManager'
+import ListView from './ListView'
+import ListViewItem from './ListViewItem'
+import ListViewItemContainer from './ListViewItemContainer'
+import ListViewItemExpansion from './ListViewItemExpansion'
+
+const stories = storiesOf('ListView', module)
+stories.addDecorator(withKnobs)
+stories.addDecorator(
+  defaultTemplate({
+    title: 'List View',
+    documentationLink: 'http://www.patternfly.org/pattern-library/content-views/list-view/'
+  })
+)
+
+stories.addWithInfo(
+  'basic list with expansion',
+  `This is the List View with basic expansion.`,
+  () => <MockListManager />
+)
+
+stories.addWithInfo(
+  'basic list with expansion knobs',
+  `This is the List View with basic expansion.`,
+  () => {
+    const firstLevelExpanded = boolean('First Level Expanded', false)
+    const secondLevelExpanded = boolean('Second Level Expanded', false)
+    return (
+      <ListView>
+        <ListViewItem isActive={firstLevelExpanded}>
+          <ListViewItemContainer
+            isExpansionItem
+            itemClicked={action('first level itemClicked')}
+            isActive={firstLevelExpanded}
+          >
+            First Level
+          </ListViewItemContainer>
+          <ListViewItemExpansion isActive={firstLevelExpanded}>
```

Comment:
@priley86 first off, i am impressed by the quality of the work done here.
in order for this work to be fully appreciated, i would like to help out in making the component as easy to use as possible.

my impression is that the building blocks of this component are very light and un-opinionated, which does make it quite flexible, and also puts the heavy lifting on the consumer side.

we may be able to make this a bit more friendly. for instance check out 
this react native implementation: 
https://github.com/naoufal/react-native-accordion

what i like here is the simplicity of usage, and still maintaining flexibility.
RN, has built in component "ListView" that accepts a json object as it's only required parameter and can make a list out of it.
the accordion in the example simply receives a header component, and a content component (which can be another accordion). 

it has more props for more advance usage, but really all that is required is header and content...

if i may suggest that maybe we can make a simple "List" component, and an "accordion" component that's built on top of it...